edgexfoundry/go-mod-core-contracts
# feat!: Allow NameFieldEscape configurable

BREAKING CHANGE: Allow NameFieldEscape configurable and add API path builder for Http and MQTT client







If your build fails due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/go-mod-core-contracts/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions

- Run unit test
- Related PRs are opened, you can refer to https://github.com/edgexfoundry/edgex-go/issues/4673#issuecomment-1709699336 and run services to verify the API.

## New Dependency Instructions (If applicable)
